app.py

```diff
@@ -871,24 +915,20 @@ def _remove_punct(tree):
 	_remove_punct(tree_copy)
 	return number_terminals(prune_empty_non_terminals(tree_copy))
 
-def tree_process(tree : ParentedTree, senttok: List[str]) -> tuple[ParentedTree, CGELTree]:
+def apply_standard_labels(tree: ParentedTree, senttok: List[str]) -> ParentedTree:
```

Comment:
Should this be a method of `ActiveDopTree`?

Reply:
Yes, I agree -- implemented in https://github.com/nschneid/activedop/pull/73/commits/655172b19c9c79a086c0776c1f6c9f0cd54229a5